examples/keyCopyExample.c

```diff
@@ -0,0 +1,70 @@
+/**
+ * @file
+ *
+ * @brief
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ */
+
+#include <kdb.h>
+#include <stdio.h>
+#include <string.h>
+
+
+void printNameAndContent (Key * k, char * varName)
+{
+	char buffer[256];
+	keyGetString (k, buffer, sizeof (buffer));
```

Comment:
keyGetString is deprecated and shouldn't be used in examples, see #3975

Any specific reason why you preferred keyGetString over keyString?

Reply:
I was not aware that it is deprecated. Thank you for pointing this out.